thrift.js

```diff
@@ -58,6 +59,46 @@ var messageExceptionTypesDef = require('./message').exceptionTypesDef;
 var validThriftIdentifierRE = /^[a-zA-Z_][a-zA-Z0-9_\.]+$/;
 
 function Thrift(options) {
+    this._init(options);
+
+    if (this.asyncLoad) {
+        return;
+    }
+
+    this._parse(this.filename, this.allowIncludeAlias);
+    this._compileAndLink();
+}
+
+// Alternative constructor allowing for asynchronous source loading.
+Thrift.load = function load(options, load, cb) {
```

Comment:
nit: Perhaps consider accepting an `fs` object directly instead of adapting `load` internally. Accepting a loader allows for the possibility of a lazy bound load implementation.

Reply:
I'm not sure I understand. Do you mean allowing to dynamically set the load function to use from the outside? I added the possibility to set `options.fs.load` instead of passing a `load` function.

Reply:
I’m thinking that it should be sufficient to accept the same `options` as the Thrift() bare constructor, including `options.fs` instead of having this `load` argument at all. One way to do things is better than two, since we don’t have to explain precedence in the doc comment.

And if `fs` is an object with a `load` method on its prototype chain, it should work just as well as an `fs` object literal with a `load` closure.

Reply:
Ok got it. I'll switch to only accepting `options.fs`, it's cleaner and more versatile indeed.

Reply:
done